### PR TITLE
Fix risk log documentation

### LINKED ISSUE #907: OSCAL documentation  wrong description for `entry` under `risk-log`
# Describe the bug
In the OSCAL documentation, the `entry` under `risk-log` is described using the same statement from `entry` under `assessment-log`

The statement used is: "Identifies the result of a task that occurred as part of executing an assessment plan or an assessment event that occurred in producing the assessment results."

# Who is the bug affecting?
All users of the documentation

# What is affected by this bug?
The reference documentation

# When does this occur?
When reading it ...

# How do we replicate the issue?
go to: [https://pages.nist.gov/OSCAL/documentation/schema/assessment-results-layer/assessment-results/xml-schema/#global_risk](https://pages.nist.gov/OSCAL/documentation/schema/assessment-results-layer/assessment-results/xml-schema/#global_risk) or to its JSON counterpart.

### PR DESCRIPTION
**Pull request by david-waltermire-nist**
# Committer Notes

Adjusting documentation for the risk log, which was copied from the assessment log and never updated to reflect the risk log semantics.

Resolves #907.

### All Submissions:

- [x] Have you followed the guidelines in our [Contributing](https://github.com/usnistgov/OSCAL/blob/master/CONTRIBUTING.md) document?
- [x] Have you checked to ensure there aren't other open [Pull Requests](https://github.com/usnistgov/OSCAL/pulls) for the same update/change?
- [x] Have you squashed any non-relevant commits and commit messages? \[[instructions](https://git-scm.com/book/en/v2/Git-Tools-Rewriting-History)\]
- [x] Do all automated CI/CD checks pass?

### Changes to Core Features:

- [x] Have you added an explanation of what your changes do and why you'd like us to include them?
- [x] Have you written new tests for your core changes, as applicable?
- [x] Have you included examples of how to use your new feature(s)?
- [x] Have you updated all [OSCAL website](https://pages.nist.gov/OSCAL) and readme documentation affected by the changes you made? Changes to the OSCAL website can be made in the docs/content directory of your branch.

### CODE REVIEW

File: src/metaschema/oscal_assessment-common_metaschema.xml [hunk 1]
```diff
@@ -1209,7 +1209,7 @@
                         </remarks>
                      </field>
                      <define-assembly name="related-response" max-occurs="unbounded">
-                        <formal-name>Action Reference</formal-name>
+                        <formal-name>Risk Response Reference</formal-name>
```
Comment (iMichaela) [inline src/metaschema/oscal_assessment-common_metaschema.xml:1212]: Line 1172 indicates that the **Risk Log Entry**: "_Identifies the result of a task that occurred as part of managing an identified risk._" <-- this is equivalent to: "_Identifies an individual risk response that occurred as part of managing an identified risk_"
So maybe line 1213 should change from: "_Identifies an individual risk response that this log entry is for._" (please note the similarities with line 1172_equivalent) to saying this is a reference related to the log entry. Proposed description for the **Risk Response Reference**: "_Identifies a reference related to the individual risk response associated with this log entry._"

**Review by iMichaela — DISMISSED** [review]
The changes look great. I have one suggestion for a possible enhancement - see comment line 1212. If @david-waltermire-nist does not think the clarification is necessary, then I am OK with the PR as is.

File: src/metaschema/oscal_assessment-common_metaschema.xml (hunk 1 above)
Comment (david-waltermire-nist) [reply src/metaschema/oscal_assessment-common_metaschema.xml:1212]: Done.

File: src/metaschema/oscal_assessment-common_metaschema.xml [hunk 2]
```diff
@@ -1169,29 +1169,29 @@
             <model>
                <define-assembly name="entry" min-occurs="1" max-occurs="unbounded">
                   <formal-name>Risk Log Entry</formal-name>
-                  <description>Identifies the result of a task that occurred as part of executing an assessment plan or an assessment event that occurred in producing the assessment results.</description>
+                  <description>dentifies an individual risk response that occurred as part of managing an identified risk.</description>
```
Comment (iMichaela) [inline src/metaschema/oscal_assessment-common_metaschema.xml:1172]: Typo here. 'dentifies' should read 'Identifies'

File: src/metaschema/oscal_assessment-common_metaschema.xml (hunk 1 above)
Comment (iMichaela) [reply src/metaschema/oscal_assessment-common_metaschema.xml:1212]: I believe the information is not clear. If my suggested text above was not appropriate, we need to find one that is. The existing <description>_Identifies an individual risk response that this log entry is for_.</description> is confusing to me. So a **Log Entry** logs a response (as part of managing risk), and it has a **Risk Response Reference**, that per <description>identifies the response the Log Entry document, so it identifies itself? I still think the description should read: <description>Identifies **a reference related to** the individual risk response associated with this log entry.</description>

Comment (david-waltermire-nist) [reply src/metaschema/oscal_assessment-common_metaschema.xml:1212]: I used your text suggestion. Although I missed the leading 'I'. I'll fix this.

File: src/metaschema/oscal_assessment-common_metaschema.xml (hunk 2 above)
Comment (iMichaela) [reply src/metaschema/oscal_assessment-common_metaschema.xml:1172]: :) Not a big problem. Just misunderstanding. We should talk to clarify - the comment and text suggested were for the Risk Response Reference (line 1213): and the proposed <description>Identifies a reference related to the individual risk response associated with this log entry.</description>. The justification for the change was using the log entry description to highlight the issue.

**Pull request merged**